packages/web/src/builders/package/package.impl.ts

```diff
@@ -134,24 +152,26 @@ export function run(
       if (options.watch) {
         return new Observable<BuildResult>(obs => {
           const watcher = rollup.watch(rollupOptions);
-          watcher.on('event', ({ code, error }) => {
-            if (code === 'START') {
+          watcher.on('event', data => {
+            if (data.code === 'START') {
               context.logger.info('Bundling...');
-            } else if (code === 'END') {
-              updatePackageJson(options, context, target, dependencies);
+            } else if (data.code === 'END') {
+              updatePackageJson(
+                options,
+                context,
+                target,
+                dependencies,
+                packageJson
+              );
               context.logger.info(
                 'Bundle complete. Watching for file changes...'
               );
               obs.next({ success: true });
-            } else if (code === 'ERROR') {
-              context.logger.error(`Error during bundle: ${error.message}`);
-              obs.next({ success: false });
-            } else if (code === 'FATAL') {
-              // Cannot continue, stop the observable.
+            } else if (data.code === 'ERROR') {
               context.logger.error(
-                `Fatal error during bundle: ${error.message}`
+                `Error during bundle: ${data.error.message}`
               );
-              obs.complete();
+              obs.next({ success: false });
```

Comment:
nice! does this mean that the watcher will continue to...watch, even if there's been an error now?

Reply:
Yup! Rollup removed the `FATAL` event, and instead now emits `ERROR` and will continue watching. https://github.com/rollup/rollup/commit/18d1aede0affda09513343a0c18127446352b8f6#diff-093a6241a0003efb710688c0b3332f53